backend/src/schema/resolvers/helpers/generateInviteCode.js

```diff
@@ -0,0 +1,5 @@
+export default function generateInviteCode() {
+  return Array.from({ length: 6 }, (n = Math.floor(Math.random() * 36)) => {
+    return String.fromCharCode(n > 9 ? n + 55 : n + 48)
```

Comment:
Here a comment would be awesome ;). Took me a sec to understand.
Also `55` is incorrect - should be `65`. See https://en.wikipedia.org/wiki/ASCII

```suggestion
    // Shift ASCII Characters to numbers (48-57) or A-Z (65-90)
    return String.fromCharCode(n > 9 ? n + 65 : n + 48)
```

Reply:
No. It is `return String.fromCharCode(n > 9 ? n + 55 : n + 48)` because 10 + 55 = 65

Reply:
But I agree in commenting this.